packages/demo/src/app/connected/(tools)/TimeLockedTransfer/page.tsx

```diff
@@ -0,0 +1,224 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { TextInput } from "@/src/components/Input";
+import { Button } from "@/src/components/Button";
+import { Textarea } from "@/src/components/Textarea";
+import { bytesConcat, ccc } from "@ckb-ccc/connector-react";
+import { Since, numFrom, numToBytes } from "@ckb-ccc/core";
```

Comment:
Please import CCC-related methods from the same package and use the `ccc` prefix for all methods to keep consistent.